pages/onboarding/index.tsx

```diff
@@ -15,8 +67,77 @@ const Onboarding: NextPage = () => {
           </Button>
         </Link>
       </Box>
+
+      <form onSubmit={handleSubmit}>
+        <TextField
+          label="Bio"
+          name="bio"
+          value={formData.bio}
+          onChange={handleChange}
+          multiline
+          rows={4}
+          fullWidth
+          inputProps={{ maxLength: 1000 }}
+          required
```

Comment:
all these fields are required, but i'm not convinced they should be? e.g. why should a user be _required_ to enter a twitter username?

Reply:
makes sense, I think by that logic none should be required?

Reply:
yup - though we should ensure that at least _one_ field is entered before submitting.  i.e. the submit button should be disabled until they've typed in something for at least one field

Reply:
is that for any specific reason?

Reply:
just better UX - no point in allowing them to submit zero information